CockroachDB TruncateAll() failure

TruncateAll() in cockroach.go does a query 
```
"select table_name from information_schema.tables where table_schema = ?;"
```
to gather table names and then joins all those table names for truncation
```
tx.RawQuery(fmt.Sprintf("truncate %s cascade;", strings.Join(table_names, ", "))).Exec()
```

The problem occurs when there are actually no tables in the output of select. By default even the empty "coke" application will not run tests because the query for truncation would become:
```
level=debug msg="truncate  cascade;"
```
which is an invalid one and it crashes:
```
created database coke_test
dumped schema for coke_development
loaded schema for coke_test
INFO[0000] go test -p 1 bitbucket.org/dainiookas/coke bitbucket.org/dainiookas/coke/actions bitbucket.org/dainiookas/coke/grifts bitbucket.org/dainiookas/coke
/models
?       bitbucket.org/dainiookas/coke   [no test files]
level=debug msg="select table_name from information_schema.tables where table_schema = $1;" $1=coke_test
[]
level=debug msg="truncate  cascade;"
--- FAIL: Test_ActionSuite (0.01s)
    --- FAIL: Test_ActionSuite/Test_HomeHandler (0.00s)
        model.go:26:
                        Error Trace:    model.go:26
                                                                suite.go:61
                                                                suite.go:88
                        Error:          Received unexpected error:
                                        pq: relation "cascade" does not exist
                                        github.com/gobuffalo/pop.(*Connection).timeFunc
                                                /Users/dainius/Documents/Go/src/github.com/gobuffalo/pop/connection.go:210
                                        github.com/gobuffalo/pop.(*Query).Exec
                                                /Users/dainius/Documents/Go/src/github.com/gobuffalo/pop/executors.go:21
                                        github.com/gobuffalo/pop.(*cockroach).TruncateAll
                                                /Users/dainius/Documents/Go/src/github.com/gobuffalo/pop/cockroach.go:238
                                        github.com/gobuffalo/pop.(*Connection).TruncateAll
                                                /Users/dainius/Documents/Go/src/github.com/gobuffalo/pop/connection.go:202
                                        github.com/gobuffalo/suite.(*Model).SetupTest
                                                /Users/dainius/Documents/Go/src/github.com/gobuffalo/suite/model.go:25
                                        github.com/gobuffalo/suite.(*Action).SetupTest
                                                /Users/dainius/Documents/Go/src/github.com/gobuffalo/suite/suite.go:61
                                        github.com/stretchr/testify/suite.Run.func2
                                                /Users/dainius/Documents/Go/src/github.com/stretchr/testify/suite/suite.go:88
                                        testing.tRunner
                                                /usr/local/Cellar/go/1.10.2/libexec/src/testing/testing.go:777
                                        runtime.goexit
                                                /usr/local/Cellar/go/1.10.2/libexec/src/runtime/asm_amd64.s:2361
                        Test:           Test_ActionSuite/Test_HomeHandler
FAIL
```

--- a/cockroach.go
+++ b/cockroach.go
@@ -229,6 +229,10 @@ func (p *cockroach) TruncateAll(tx *Connection) error {
 		return err
 	}
 
+	if len(tables) == 0 {
+		return nil
+	}
+
 	table_names := make([]string, len(tables))
 	for i, t := range tables {
 		table_names[i] = t.TableName

